Hax/css/screen.css

```diff
@@ -328,6 +328,15 @@ p code {
   color: #fff;
 }
 
+.branding .logo-askew {
+  animation-name: normal;
```

Comment:
That strikes me as the kind of thing a minify process could do better than a human. ^_^